src/index.html

```diff
@@ -20,8 +20,312 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+          width="40"
+          height="40"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Smartphones</a></li>
+          <li>
+            <a
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li><a href="#">Gadgets</a></li>
+          <li><a href="#">Tablets</a></li>
+          <li><a href="#">Photo</a></li>
+          <li><a href="#">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product Image"
+          class="card__image"
+        />
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <p class="card__price">
+          <span class="card__price-label">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </p>
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards 
```suggestion
      </div>
      
      <div class="card">
```